module-2/backend/week_two.md

```diff
@@ -26,20 +34,50 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+Owner.where(team_id: 4)
+
 5. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+Could be one to many or many to many depending on the type of school and grade level. 
+
+One to many: teacher will have many students, students belong to a teacher. Students have a teacher id along with the other attributes they have without the associations.
+
 6. Define foreign key, primary key, and schema.
+Foreign key of teacher id lives on student and the primary key would be that.
+
 7. Describe the relationship between a foreign key on one table and a primary key on another table.
+Foreign key lives on the table that belongs to the table that has the primary key.
+
 8. What are the parts of an HTTP response?
+A request is sent to the server which then references the corresponding controller action, which then reaches into the model and interaction with ORM, which returns the info to the controller so then the appropriate view (string) response can be sent to the client.
```

Comment:
This is describing the request/response cycle, but we wanted the actual parts of an HTTP response:

* Status Line
* Headers
* Body